iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/NetworkImpl.java

```diff
@@ -974,6 +974,7 @@ private void replaceDanglingLineByLine(List<MergedLine> lines, Multimap<Boundary
                     mergedLine.id, mergedLine.xnode, mergedLine.country1, mergedLine.country2);
             TieLineAdderImpl la = newTieLine()
                     .setId(mergedLine.id)
+                    .setName(mergedLine.half1.name + " + " + mergedLine.half2.name)
```

Comment:
As mergedLine.half1.name and mergedLine.half2.name could be null, this merged line name could end up with a name like "null + AAAA". Use Objects.toString(..., "") to avoid this.